### PR TITLE
Fix dev version generation in setup.

### PR DESCRIPTION
**Pull request by saimn**
The current commit hash/number detection was broken.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,16 +23,17 @@
     command_number = 'git rev-list --count HEAD'
 
     try:
-        commit_hash = subprocess.check_output(command, shell=True).decode('ascii').strip()
-        commit_number = subprocess.check_output(command, shell=True).decode('ascii').strip()
+        commit_hash = subprocess.check_output(command_hash, shell=True)\
+            .decode('ascii').strip()
```
Comment (astrofrog) [inline setup.py]: I this this is a case where breaking the line doesn't help readability - could you remove the line break? (also in the line below)

File: setup.py [hunk 2]
```diff
@@ -23,16 +23,17 @@
...
     except Exception:
         pass
     else:
-
         # We write the git hash and value so that they gets frozen if installed
         with open(os.path.join('glue', '_githash.py'), 'w') as f:
-            f.write("__githash__ = \"{githash}\"\n".format(githash=commit_hash))
-            f.write("__dev_value__ = \"{dev_value}\"\n".format(dev_value=commit_number))
+            f.write("__githash__ = \"{}\"\n".format(commit_hash))
+            f.write("__dev_value__ = \"{}\"\n".format(commit_number))
```
Comment (astrofrog) [inline setup.py]: These changes aren't really necessary - can you revert them? (also, ``{}`` won't work on Python 2.6)

**Comment by astrofrog** [comment]
I've implemented the minor changes and opened a new pull request (that includes your commit): https://github.com/glue-viz/glue/pull/720

**Comment by saimn** [comment]
Thanks for having made the changes to merge this Thomas, I was in vacation ;-)

File: setup.py (hunk 1 above)
Comment (ChrisBeaumont) [inline setup.py]: I tend to format long method chains like this:

```
commit_hash = (subprocess
               .check_output(command_hash, shell=True)
               .decode('ascii')
               .strip()
)
```

**Pull request closed without merging**